src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -511,22 +479,109 @@ impl<'b> Resolver<'b> {
         }
         module.populated.set(true)
     }
+
+    // does this attribute list contain "macro_use"?
+    fn contains_macro_use(&mut self, attrs: &[ast::Attribute]) -> bool {
+        for attr in attrs {
+            if attr.check_name("macro_escape") {
```

Comment:
Ugh, I haven't noticed it's `span_warn` and not `span_err`, even the variable is named err...